tools/update-shorthands/src/assignShorthandPriority.ts

```diff
@@ -0,0 +1,18 @@
+import { type CSSShorthands } from './types';
+
+export function assignShorthandPriority(shorthandProperties: CSSShorthands): CSSShorthands {
+  const propertiesWithPriority = { ...shorthandProperties };
+
+  Object.keys(shorthandProperties)
```

Comment:
maybe assign the default `-1` priorities in this function? The fact that the default priorities were assigned in the filter function confused me

Reply:
Makes sense, will redo it.